docs/conceptual_guides/quantization_schemes.md

```diff
@@ -0,0 +1,88 @@
+# Quantization Schemes
+
+Quantization is a technique to reduce the computational and memory costs of running inference by representing the weights and activations with low-precision data types like 8-bit integer (`int8`) instead of the usual 16-bit floating point (`float16`).
+
+## Theory
+
+Performing quantization to go from `float16` to `int8` (or lower) is tricky. Only 256 values can be represented in `int8`, while `float16` can represent a very wide range of values. The idea is to find the best way to project our range [a, b] of `float32` values to the `int8` space.
+
+Let’s consider a float x in [a, b], then we can write the following quantization scheme:
+
+```bash
+x = S * (x_q - Z)
+```
+
+where:
+
+- `x_q` is the quantized `int8` value associated to `x`
+- `S` is the scale, and is a positive `float16`
+- `Z` is called the zero-point, it is the `int8` value corresponding to the value 0 in the `float16` realm. If zero-point is ommited, we call this "symmetric" quantization.
+
+
+The quantized value x_q of x in [a, b] can be computed as follows:
+
+```bash
+x_q = round(x/S + Z)
+```
+
+And `float16` values outside of the [a, b] range are clipped to the closest representable value, so for any floating-point number x:
+
+```bash
+x_q = clip(round(x/S + Z), round(a/S + Z), round(b/S + Z))
+```
+
+## Quantization Flavors
+
+There are several flavors of quantization.
+
+### Static vs Dynamic
+
+The section above described how quantization from `float16` to `int8` works, but did not explain how to compute the scales and zero points.
+
+With weights, since the full range is known ahead of time, we can just compute the scales and zero points statically (sometimes using a more sophisticated algorithm like `GPTQ`).
+
+With activations, however, there are two approaches:
+* **Dynamic quantization**: the range for each activation is computed on the fly at runtime. This gives us the best possible values, but it can be a bit slower than static quantization because of the overhead introduced by computing the range each time. It is also not an option on certain hardware.
```

Comment:
```suggestion
* **Dynamic quantization**: the range for each activation is computed on the fly at runtime so that the quantization range matches exactly the current runtime range. This gives us the best possible values, but it can be a bit slower than static quantization because of the overhead introduced by computing the range each time. It is also not an option on certain hardware.
```